testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -277,6 +277,22 @@ def forward(self, image):
 resize = nn.Upsample(size=[6, 8], mode='bilinear')
 save_data_and_model("resize_bilinear", input, resize)
 
+input = Variable(torch.randn(1, 3, 4, 5))
+upsample_unfused = nn.Sequential(
+        nn.Conv2d(3, 2, kernel_size=3, stride=1, padding=1),
+        nn.BatchNorm2d(2),
+        nn.Upsample(scale_factor=2, mode='nearest')
+        )
+save_data_and_model("upsample_unfused", input, upsample_unfused)
+
+input = Variable(torch.randn(1, 3, 4, 5))
+resize_nearest_unfused = nn.Sequential(
+        nn.Conv2d(3, 2, kernel_size=3, stride=1, padding=1),
+        nn.BatchNorm2d(2),
+        nn.Upsample(scale_factor=2, mode='nearest')
+        )
+save_data_and_model("resize_nearest_unfused", input, resize_nearest_unfused, 11)
```

Comment:
Please updage generator with `resize_nearest_unfused1` and `resize_nearest_unfused2`

Maybe let's rename them to `resize_nearest_unfused_opset_11` or something like that?

Reply:
Yeah, it would be better and what about pytorch version ?

Reply:
@dkurt  `resize_nearest_unfused1` and `resize_nearest_unfused2` both are exported using opset version 11. The difference is in pytorch version. `resize_nearest_unfused1` is pytorch version 1.3 and `resize_nearest_unfused2` is pytorch version 1.4.

`upsample_unfused` is exported using opset version 9.

Not sure if `resize_nearest_unfused_opset_11_torch_1.3` and `resize_nearest_unfused_opset_11_torch_1.4` would be okay. And both cannot be exported together because of pytorch version difference. How should I add both of them in generator because code is exact same, just pytorch version difference ?

Reply:
Got it, thanks! Yeah, let's name it with version if PyTorch and opset. Just not ti forget :)

Reply:
Done !